code/game/objects/items/crayons.dm

```diff
@@ -105,7 +105,7 @@
 	return
 
 /obj/item/pen/crayon/attack(mob/M as mob, mob/user as mob, var/target_zone)
-	if(M == user)
+	if(M == user && M.check_has_mouth())
 		to_chat(user, "You take a bite of the crayon and swallow it.")

```

Comment:
Might be worth adding visible messages to that, so the people around can "see" them eating the crayon in the chat.